daemon/web/common/AuthService.js

```diff
@@ -0,0 +1,38 @@
+
+
+const TOKEN_KEY = 'inertia';
+
+export const login = () => {
+  // set jwt
+};
+
+export const logout = () => {
+  // clear jwt
+};
+
+export const isAuthenticated = () => {
+  // verifies that jwt is still valid
+  return true;
+};
+
+export const guardRoute = () => {
+  if (!isAuthenticated()) {
+    // push route
+  }
+};
+
+export const setToken = (token) => {
```

Comment:
The daemon should be setting cookies itself - I don't think you have to manually manage tokens.

[Setting cookie upon login](https://github.com/ubclaunchpad/inertia/blob/rob/%23127-authentication/daemon/inertia/auth/sessions.go#L90)

[Removing cookies](https://github.com/ubclaunchpad/inertia/blob/rob/%23127-authentication/daemon/inertia/auth/sessions.go#L122)

[Tests verify that cookies are received (I hope)](https://github.com/ubclaunchpad/inertia/blob/rob/%23127-authentication/daemon/inertia/auth/permissions_test.go#L145)

Reply:
Ah i see, I was wondering how we are handling this.